tests/viscoelastic_bending_beam.prm

```diff
@@ -26,3 +26,27 @@ subsection Postprocess
     set Output format = txt
   end
 end
+
+# Number and name of compositional fields
+subsection Compositional fields
+  set Number of fields = 4
+  set Names of fields  = ve_stress_xx, ve_stress_yy, ve_stress_xy, beam
+  set Types of fields  = stress, stress, stress, chemical composition
+end
+
+
+# Material model
+subsection Material model
+
+  set Model name = viscoelastic
+
+  subsection Viscoelastic
+    set Densities            =  2800,  3300
+    set Viscosities          = 1.e18, 1.e24
+    set Elastic shear moduli = 1.e11, 1.e11, 1.e11, 1.e11, 1.e10
```

Comment:
I know, this is annoying. I left it because if I change it this pr gets larger and more difficult to review. But I'm happy to add it to the mods if you want.